packages/get-platform/src/__tests__/getSSLVersion.test.ts

```diff
@@ -5,8 +5,9 @@ const describeIf = (condition: boolean) => (condition ? describe : describe.skip
 describeIf(process.platform === 'linux')('getSSLVersion', () => {
   it('should not return an error', async () => {
     const arch = 'x64'
-    await getSSLVersion({ targetDistro: 'debian', arch })
-    await getSSLVersion({ targetDistro: 'musl', arch })
-    await getSSLVersion({ targetDistro: 'rhel', arch })
+    const archFromUname = 'x86_64'
+    await getSSLVersion({ targetDistro: 'debian', arch, archFromUname })
+    await getSSLVersion({ targetDistro: 'musl', arch, archFromUname })
+    await getSSLVersion({ targetDistro: 'rhel', arch, archFromUname })
```

Comment:
The test was passing, but it's not type-checked, so we had forgot to pass one argument